test/cbmc/stubs/core_sntp_stubs.c

```diff
@@ -77,11 +78,20 @@ int32_t NetworkInterfaceReceiveStub( NetworkContext_t * pNetworkContext,
     if( tries < ( MAX_NETWORK_RECV_TRIES - 1 ) )
     {
         tries++;
+
+        if( bytesOrError > 0 )
+        {
+            read += bytesOrError;
+        }
     }
     else
     {
         tries = 0;
-        bytesOrError = 0;
+
+        /* This ensures that all the remaining bytes are received in the last try
+         *  if there are any bytes yet to be received and covers the case that SntpSuccess
```

Comment:
(Minor) Formatting issue

Reply:
Can you also put a disclaimer that it is possible for this logic to return a negative value if the accumulated value of `read` over the multiple tries is greater than `SNTP_PACKET_BASE_SIZE`